sandbox.go

```diff
@@ -691,6 +691,9 @@ func (sb *sandbox) EnableService() error {
 func (sb *sandbox) DisableService() error {
 	logrus.Debugf("DisableService %s START", sb.containerID)
 	for _, ep := range sb.getConnectedEndpoints() {
+		if err := ep.deleteServiceInfoFromCluster(sb, "DisableService"); err != nil {
```

Comment:
This logic was intentionally removed to avoid a race with the sbLeave. So far the sbLeave is the only code path that actually cleans the name resolution/load balancer.